docs/sigs/HPC and AI.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "HPC and AI SIG"
+---
+# HPC and AI SIG
+
+AlmaLinux is the ideal Linux choice for HPC and AI applications, and the HPC and AI SIG is here to help you make the most of it. We are a group of HPC and AI experts who use AlmaLinux to power our projects and share our knowledge with the AlmaLinux community. We promote the benefits of AlmaLinux for HPC and AI, provide guidance and support for AlmaLinux users in HPC and AI, and represent the interests and needs of the HPC and AI community within AlmaLinux. Join us today and discover how AlmaLinux can boost your HPC and AI performance and productivity.
+
+## How to Join
+
+Joining is easy! You can show up to a meeting, pick up an issue from the list by assigning it to yourself, or ask questions in chat! Not every contributor wants to be a part of the SIG, but if you do, joining is simple. 
+
+**Where we chat**
+
+* [SIGHPCAI](https://chat.almalinux.org/almalinux/channels/sighpcai) on [chat.almalinux.org](https://chat.almalinux.org)
+* [HPC and AI on the AlmaLinux Forum](https://almalinux.discourse.group/c/sigs/hpc/31)
+* [HPC and AI Mailing List](https://lists.almalinux.org/postorius/lists/hpcai.lists.almalinux.org/)
+
+**Where and when we meet**
+
+The HPC and AI SIG gathers quarterly as a group for open discussion. The HPC and AI SIG also occasionally presents on HPC and AI-related topics and hosts community gatherings at HPC and AI-related events. Quarterly meetings and other events are announced via the [HPC and AI Mailing List](https://lists.almalinux.org/postorius/lists/hpcai.lists.almalinux.org/).
+
+## Activities, projects, and deliverables
+
+* AlmaLinux System Series guidance on deploying AlmaLinux in HPC and AI environments.
+* Example configuration files for deploying Slurm and other HPC software on AlmaLinux: [AlmaLinux/sig-hpcai](https://github.com/AlmaLinux/sig-hpcai).
+
+### Help wanted
+
+Contributions take all kinds of shapes and sizes, and we welcome everyone! If you have expertise or time and would like to help us out, we welcome contributions to our activities, projects, and deliverables above. We also welcome community members to present on topics related to their use of AlmaLinux for HPC or AI.
+
+## SIG members
+
+* [Hayden Barnes](mailto:hayden.barnes@hpe.com) - AI Open Source Community Manager, [HPE](https://www.determined.ai/)
```

Comment:
Is the link correct here? Alt text is HPE but it's pointing to `https://www.determined.ai`?